commands/CLI/test.go

```diff
@@ -0,0 +1,25 @@
+package team23
+
+import (
+	"fmt"
+	//"github.com/emilejbm/461_1_23/functionality/CLI"
+	"github.com/spf13/cobra"
+)
+
+var testCmd = &cobra.Command{
```

Comment:
I would change the filename. Maybe start organizing the test suite folder and structure?